job-dsl-plugin/src/main/groovy/javaposse/jobdsl/plugin/casc/SeedJobConfigurator.java

```diff
@@ -10,7 +10,10 @@
 import io.jenkins.plugins.casc.impl.attributes.MultivaluedAttribute;
 import io.jenkins.plugins.casc.model.CNode;
 import io.jenkins.plugins.casc.model.Mapping;
-import java.util.Map;
+
+import java.util.*;
```

Comment:
apart from the star import, this looks good to me!

Reply:
Oops, my IDE did that and I didn't notice :(

Reply:
Yes, please remove the star import and keep the existing order.

Reply:
Done.